public/js/SceneManager.js

```diff
@@ -299,12 +272,18 @@ class SceneManager2 {
 
     this.initiallyPaused = false;
     this.sceneEntities = [];
-    this.sceneEntities.push(new TextScreen(this.game, LEVEL_ONE_TEXT));
-    this.sceneEntities.push(new Overlay(this.game, true, LEVEL_TRANSITION_OVERLAY_TIME));
+    this.sceneEntities.push(new TextScreen(this.game, smc.LEVEL_ONE_TEXT));
+    this.sceneEntities.push(new Overlay(this.game, true, smc.LEVEL_TRANSITION_OVERLAY_TIME));
     this.startedFinalOverlay = false;
```

Comment:
Yes! thanks for moving these, I have a couple more for the boss death, but I will take care of those.